{{cookiecutter.project_slug}}/ios/Podfile

```diff
@@ -0,0 +1,14 @@
+# Uncomment the next line to define a global platform for your project
```

Comment:
Please rebase with master so you can have access to our last version of podfile

Reply:
Got it